packages/calcite-components/src/components/slider/slider.scss

```diff
@@ -1,27 +1,54 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-slider-fill-color: Specifies the color of the fill.
+ * @prop --calcite-slider-graph-accent-color: Specifies the accent color of the graph.
+ * @prop --calcite-slider-graph-background-color: Specifies the background color of the graph.
+ * @prop --calcite-slider-handle-border-color-hover: Specifies the border color of the handle when hovered.
+ * @prop --calcite-slider-handle-border-color: Specifies the border color of the handle.
+ * @prop --calcite-slider-handle-color-active: Specifies the color of the handle.
+ * @prop --calcite-slider-handle-color-hover: Specifies the color of the handle when hovered.
+ * @prop --calcite-slider-handle-color: Specifies the color OF the handle.
+ * @prop --calcite-slider-handle-extension-color-hover: Specifies the color of the handle extension when hovered.
+ * @prop --calcite-slider-handle-extension-color: Specifies the color of the handle extension.
```

Comment:
Maybe this could use the `precise` nomenclature that displays this? Either in prop name or description.